src/Microsoft.Extensions.Hosting/Internal/ConsoleLifetime.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Hosting.Internal
+{
+    /// <summary>
+    /// Listens for Ctrl+C or SIGTERM and initiates shutdown.
+    /// </summary>
+    public class ConsoleLifetime : IHostLifetime
+    {
+        public ConsoleLifetime(IHostingEnvironment environment, IApplicationLifetime applicationLifetime)
+        {
+            Environment = environment ?? throw new ArgumentNullException(nameof(environment));
+            ApplicationLifetime = applicationLifetime ?? throw new ArgumentNullException(nameof(applicationLifetime));
+            ApplicationLifetime.ApplicationStarted.Register(() =>
+            {
+                Console.WriteLine("Application started. Press Ctrl+C to shut down.");
```

Comment:
This shouldn't be in our built-in ConsoleLifetime.

Reply:
Discussed earlier with fowler. This is existing behavior that has been consolidated to an appropriate place. If you're not using console it goes away. We can also give ConsoleOptions to control it. Note this used to be built into RunAsync, without it you don't get any startup output now.

Reply:
Fair, but definitely give me an option. I want to be able to use this for command line tools (I have some design thoughts for that, but they're all out of scope for our built in components) and I definitely don't want this printed out in those cases.